functions/responsestreaming/StreamBigQuery.Tests/StreamBigQuery.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I'd expect to see a solution file containing both project files, along with a `runTests.ps1` file - please make this directory look like all the other ones, basically.